From ec477f05839e1850b8b8aea5f53189ce90b26e4f Mon Sep 17 00:00:00 2001
From: Aleksey Lim <alsroot@member.fsf.org>
Date: Sat, 3 Apr 2010 13:27:35 +0000
Subject: Do not use deprecated API #1899


diff --git a/gtk/engine/sugar-info.c b/gtk/engine/sugar-info.c
index b4bf9c5..8d7832f 100644
--- a/gtk/engine/sugar-info.c
+++ b/gtk/engine/sugar-info.c
@@ -71,7 +71,7 @@ sugar_fill_range_info (SugarRangeInfo *range_info, gboolean trough)
     gdouble line_width = info->rc_style->line_width;
 
     if (info->widget) {
-        range_info->focused = GTK_WIDGET_HAS_FOCUS (info->widget);
+        range_info->focused = gtk_widget_has_focus (info->widget);
     } else {
         /* Fall back to unfocused and orientation from the width/height. */
         range_info->focused = FALSE;
diff --git a/gtk/engine/sugar-style.c b/gtk/engine/sugar-style.c
index b81fe69..88a80a4 100644
--- a/gtk/engine/sugar-style.c
+++ b/gtk/engine/sugar-style.c
@@ -415,7 +415,7 @@ sugar_style_draw_box (GtkStyle       *style,
         sugar_draw_button (cr, &info);
 
         /* Spinbutton focus hack. */
-        if (widget && GTK_WIDGET_HAS_FOCUS (widget)) {
+        if (widget && gtk_widget_has_focus (widget)) {
             /* Draw a focus for the spinbutton */
             sugar_style_draw_focus (style, window, GTK_STATE_NORMAL, area, widget, detail, x, y, width, height);
         }
@@ -444,7 +444,7 @@ sugar_style_draw_box (GtkStyle       *style,
 
             sugar_fill_generic_info (&info, style, state_type, shadow_type, widget, detail, x, y, width, height);
             
-            if (widget && !GTK_WIDGET_IS_SENSITIVE (widget))
+            if (widget && !gtk_widget_is_sensitive (widget))
                 info.state = GTK_STATE_INSENSITIVE;
 
             /* Needed because the trough and bar are cached in a buffer inside GtkProgress. */
@@ -458,7 +458,7 @@ sugar_style_draw_box (GtkStyle       *style,
 
             sugar_fill_generic_info (&info, style, state_type, shadow_type, widget, detail, x, y, width, height);
             
-            if (widget && !GTK_WIDGET_IS_SENSITIVE (widget))
+            if (widget && !gtk_widget_is_sensitive (widget))
                 info.state = GTK_STATE_INSENSITIVE;
             
             if (widget && GTK_IS_PROGRESS_BAR (widget)) {
@@ -585,7 +585,7 @@ sugar_style_draw_flat_box (GtkStyle       *style,
 {
     /* Hack to change the entries background when it has the focus. */
     if (DETAIL ("entry_bg")) {
-        if (widget && GTK_WIDGET_HAS_FOCUS (widget)) {
+        if (widget && gtk_widget_has_focus (widget)) {
             state_type = GTK_STATE_ACTIVE;
         }
     } else if (DETAIL ("radiobutton") || DETAIL ("checkbutton")) {
@@ -638,10 +638,10 @@ sugar_style_draw_shadow (GtkStyle       *style,
         /* XXX: This fakes an ACTIVE state for the focused entry.
          *      Getting this changed in GTK+ with a style property would be cleaner
          *      as that also works for the font colors. (see also draw_flat_box) */
-        if (widget && GTK_WIDGET_HAS_FOCUS (widget)) {
+        if (widget && gtk_widget_has_focus (widget)) {
             info.state = GTK_STATE_ACTIVE;
         }
-        if (widget && !GTK_WIDGET_IS_SENSITIVE (widget)) {
+        if (widget && !gtk_widget_is_sensitive (widget)) {
             info.state = GTK_STATE_INSENSITIVE;
         }
 
-- 
1.7.0.2

